\documentclass[11pt]{article}
\pdfoutput=1
\usepackage{simplemargins}
\usepackage[pdftex]{graphicx}
\graphicspath{{figures/}}
\usepackage{siunitx}
\bibliographystyle{plain}
\setlength{\parindent}{0pt} 
\setlength{\parskip}{1.6ex} 
\setallmargins{1in} 
\linespread{1.6}

\begin{document}


% Title must be 150 characters or less
\begin{flushleft} 
{\Large \textbf{The genetic architecture of local adaptation I: The genomic landscape of 
foxtail pine (\textit{Pinus balfouriana} Grev. \& Balf.)}}
% Insert Author names, affiliations and corresponding author email.
\\
Christopher J.\ Friedline$^{1}$, 
Brandon M. Lind$^{1}$,
Erin M. Hobson,$^{1}$,
Douglas E. Harwood$^{1}$, 
Annette Delfino Mix$^{2}$,
Patricia E. Maloney$^{3}$, and
Andrew E. Eckert$^{1,4}$
\\
\bf{1} Department of Biology, Virginia Commonwealth University, Richmond, VA 23225
\\
\bf{2} Institute of Forest Genetics, USDA Pacific Southwest Research Station, Placerville, 
CA 95667
\\
\bf{3} Department of Plant Pathology, University of California, Davis, CA 95616
\\
\bf{4} Author for Correspondence

$\ast$ E-mail: aeckert2@vcu.edu
\end{flushleft}

\section{Abstract}

\section{Introduction}
The genetic architecture of fitness-related traits has been a major focus of geneticists for over a century. 
Genetic architecture refers to the number, type, effect size, genomic organization, 
interactions among, and environmental dependency 
of the loci contributing to the phenotypic variation which in turn creates variation in fitness among 
individuals within populations.
Interest in this architecture stems from the want to explain the nature of 
genetic variation which contributes to evolution 
via the accumulation of adaptations (i.e. adaptive evolution). Early efforts to understand 
this architecture focused primarily on the number and effect size of the loci underlying heritable, 
phenotypic variation (e.g. Fisher 1918). 
Recent work has extended this line of research, with myriad studies linking 
phenotypic with genetic variation through 
some form of linkage mapping, either within pedigrees (XXXXX) or within unstructured populations (XXXX). 
Despite great advances in experimental technology, the empirical focus has remained almost 
fully on the number, effect size, and type of loci
contributing to adaptive evolution (XXXX). The genomic organization of such loci, 
however, is relevant to much of this architecture, as 
it affects many of the other attributes of genetic architecture listed previously, 
including the ability to detect the loci as underlying trait variation. 
Any thorough examination of the genetic architecture of fitness-related traits 
therefore should include some aspect of the genomic organization of the loci
contributing to trait variation. Here, we leverage this idea in the first of a series of papers 
dissecting the genetic architecture of fitness-related
traits in a non-model conifer species, foxtail pine (\textit{Pinus balfouriana} Grev. \& Balf.).

Ideally, the genomic organization of loci contributing to variation in fitness-related traits would 
follow naturally from the production of a genome sequence (i.e. a physical map). For many taxa, especially those 
with small to modest genome sizes,
this is monetarily and computationally feasible using next-generation DNA sequencing technologies (cf. XXXXX). For taxa with large or complex
genomes, even the advent of next generation DNA sequencing makes the production of a genome sequence 
a more complicated and costly endeavor. Conifers have large and complex genomes (Murray 1998), with estimated average genome sizes in \textit{Pinus} in the range of \SIrange{20}{30}{Gbp},
which is approximately 10-fold larger than the human genome. 
Several genome projects, each involving many laboratories, are
underway or have been completed (XXXXXX). Even these efforts often initially 
result in limited information, as current assemblies of the Norway spruce and
loblolly pine genomes contain millions of unordered contigs with average sizes in the thousands of base pairs (XXXXXXX). An alternative, 
but not mutually exclusive, approach to describing the genome of an
organism is that of linkage mapping. In this approach, genetic markers are ordered through 
observations of recombination events within pedigrees. This approach
dates to the beginning of genetics and the logic has remained relatively unchanged since 
Sturtevant created the first linkage maps in \textit{Drosophila} (XXXX). 

Linkage maps have a long history within forest genetics, mostly through their use in quantitative trait locus
mapping (Ritland et al. 2011). Conifers in particular are highly amenable to linkage mapping, with approximately
25 different species currently having some form of linkage map (see Table 5-1 in Ritland et al. 2011). Much of the amenability of
conifers to linkage mapping stems from the early establishment of breeding populations in economically important species and from the presence of a 
multicellular female gametophyte (i.e. the megagametophyte) from which the haploid product of maternal meiosis can be observed (Cairney and Pullman 2007;
XXXXXX). Indeed, many of the first linkage maps in conifers were generated from collections of megagametophytes made from single trees (XXXX).
Continued development of genetic marker technologies facilitated rapid development of linkage maps across a diversity of species, with 
the largest maps generated for economically important species (e.g. XXXXXXXX). Comparisons among maps using putatively orthologous gene
sets revealed strong levels of macro-synteny across conifer clades (e.g \textit{Pinaceae}) despite their enormous genomes. This lent credence to 
the practice of linkage mapping in conifers (Neale et al. 1994), although the development of biologically informative markers was hampered by cost in species without direct 
economic importance to the wood and pulp industry. For example, the vast majority of linkage maps outside of economically important species were
created with anonymous genetic markers (e.g. XXXXX). As such, much of the knowledge about the genetic architecture of fitness-related traits outside of a
handful of economically important conifer species was about the number and effect size of uncharacterized genetic markers.

Renewed interest in linkage maps has occurred for two basic reasons. First, linkage maps can be used 
to order contigs created during genome sequencing projects (XXXXXX). In this fashion, linkage maps are
used to help create larger contigs from those generated during the assembly. It is these larger contigs that
create the utility that most practicing scientists attribute to genome sequences. Second, linkage maps are 
relatively easy to produce and provide a rich context with which to interpret
population and quantitative genetic patterns of variation (e.g. Eckert et al. 2010a,b; Eckert et al. 2013). 
They can also be used to test explicit hypotheses about the organization of loci contributing to 
adaptive evolution (XXXXX). For example, Yeaman and Whitlock (XXXX) developed theoretical predictions
about the genomic organization of loci underlying patterns of local adaptation as a function of gene 
flow, so that loci contributing to local adaptation have differing spatial structure within genomes
under different regimes of gene flow. The relevant scale (sensu Houle et al. 2011) in these mathematical formulations 
is that of recombinational distance among loci,
as it is recombination that breaks apart advantageous allelic combinations across loci, so that when 
matched with an appropriate study system, linkage maps provide the impetus to test basic 
evolutionary principles (XXXXXX). 

LAST PARAGRAPH HERE.


\section{Materials and Methods}

\subsection{Focal species}
Foxtail pine is five needle species of pine classified into 
subsection \textit{Balfourianae}, section \textit{Parrya}, and subgenus \textit{Strobus} 
(Gernandt et al. 2008). It is one of three species within subsection \textit{Balfourianae} 
(Bailey 1970) and generally is regarded as the sister species to Great Basin bristlecone pine (\textit{P. longaeva}; see 
Eckert and Hall 2006). The natural range of foxtail pine encompasses two 
regional populations located within California that are separated by approximately 500 km - 
the Klamath Mountains of northern California and the Sierra Nevada of southern California 
(Figure 1). These regional populations diverged approximately one million years ago (mya), 
with current levels of gene flow between these regional populations approximately zero 
(Eckert et al. 2008). Within each regional population, levels of genetic diversity and the 
degree of differentiation among local stands differ, with genetic diversity being highest in 
the southern Sierra Nevada population and genetic differentiation being the highest in the 
Klamath population (Oline et al. 2000; Eckert 2006; Eckert et al. 2008).

\subsection{Sampling}
Seed lots from approximately 140 maternal trees distributed throughout the natural range 
of foxtail pine were obtained during 2011 and 2012. XXX of these were sampled from the 
Klamath region, while XXX were sampled from the southern Sierra Nevada region. Approximately 
50 seeds were germinated from each seed lot and 35 of those 50 seedlings were planted in a 
common garden located at the USDA Institute of Forest Genetics, Placerville, California. The 
common garden was established using a randomized block design XXX MORE HERE. Five of the 
140 maternal trees were selected at random ($n = 3$ from the Klamath region and $n = 2$ from 
the southern Sierra Nevada) for linkage analysis. For each of these trees, \SIrange{75}{100}{} 
seeds were germinated and planted in the common garden. Upon germination, haploid 
megagametophyte tissue was rescued from each growing seedling, cleaned, and stored for further 
analysis in \SI{1.5}{\mL} Eppendorf tubes at \SI{-20}{\celsius}.


\subsection{Library Preparation and Sequencing}

Total genomic DNA was isolated from each rescued megagametophyte using the DNeasy 96 Plant 
kit following the manufacturer’s protocol (Qiagen, Germantown, MD). Restriction site associated DNA 
sequencing (RADseq, see Davey and Blaxter 2010) was used to generate a genome-wide set of 
single nucleotide polymorphism (SNP) markers for linkage mapping following the protocol 
outlined by Parchman et al. (2012). In brief, this protocol is a double digestion RADseq 
approach based on digestion of total genomic DNA using EcoR1 and MseI followed by single-end 
sequencing on the Illumina HiSeq 2000 (or 2500) platform. Following digestion, adaptors 
containing amplification and sequencing primers, as well as barcodes for multiplexing, 
were ligated on to digested DNA fragments. Here, we chose to multiplex 96 samples using the 
barcodes available from Parchman et al. (2012). These barcodes are a mixture of 8, 9, and 
10 bp tags that differ by at least four bases, so as to accommodate sample identification in the 
presence of sequencing errors. Following ligation, successfully ligated DNA fragments were 
amplified using PCR and amplified fragments were size selected using gel electrophoresis. We selected 
fragments in the size range of 400 bp (\SIrange{300}{500}{bp}) by excising and purifying DNA from 2.5\% 
agarose gels using QIAquick Gel Extraction Kits (Qiagen).


\subsection{DNA Sequence Analysis}

\subsection{Linkage Analysis}

\section{Results}

\section{Discussion}

\section{Acknowledgements}

The authors would like to thank the staff at the USDA Institute of Forest Genetics, the 
VCU Nucleic Acids Research Facility, and the VCU Center for High Performance Computing. 
In addition, we would like to thank Tom Blush and XXX Tricia’s husband for help in obtaining 
seeds. Funding for this project was made available to AJE via startup funds from Virginia 
Commonwealth University. CJF was supported under NSF NPGI: Postdoctoral Research Fellowship 
in Biology FY 2013 Award \#IOS-1306622.

\end{document}
